NeoSwift/NeoClient.swift

```diff
@@ -37,11 +61,12 @@ public class NeoClient {
         self.seed = seed
     }
     
-    func sendRequest(_ method: RPCMethod, params: [Any]?, completion :@escaping ([String:AnyObject]?, Error?) -> Void) {
+    func sendRequest(_ method: RPCMethod, params: [Any]?, completion: @escaping (NeoClientResult<JSONDictionary>) -> ()) {
         guard let url = URL(string: seed) else {
-            completion(nil, "Invalid Seed" as? Error)
+            completion(.failure(NeoClientError.invalidSeed))
             return
```

Comment:
can this be (.failure(.invalidSeed))

Reply:
Yep can be done, but then we will only return `NeoClientError` error type.